automation/check-patch.setup.sh

```diff
@@ -25,3 +25,7 @@ cp -rf $(pwd)/. ${TMP_PROJECT_PATH}
 
 echo 'Exporting temporary project path'
 export TMP_PROJECT_PATH
+
+echo 'Exporting KubeMacPool component project path'
+KUBEMACPOOL_TMP_PROJECT_PATH=${TMP_PROJECT_PATH}/_components/go/src/github.com/k8snetworkplumbingwg/kubemacpool/
```

Comment:
Also, you can't depend on _components, you don't create it here. If you want to use kubemacpool sources here, we need to introduce a way to fetch it explicitly